hg-fast-export.py

```diff
@@ -161,7 +161,15 @@ def export_file_contents(ctx,manifest,files,hgtags,repourl,revnode,ignoreSub,enc
     if not hgtags and file == ".hgtags":
       sys.stderr.write('Skip %s\n' % (file))
       continue
+    # Skip .git files. They only get us in trouble.
+    if '.git' in file:
+      sys.stderr.write('Skip %s\n' % (file))
+      continue
     d=ctx.filectx(file).data()
+    # Skip files bigger than 3 MB
+    if len(d) > 3145728:
```

Comment:
I will remove the last two commits and put them on a separate branch
As soon as git LFS is more stable I plan to rely on it for binary files
I'll try to add a parameter as you suggested but I am neither a python nor a fast-export expert ;)